builders/pallets-precompiles/precompiles/randomness.md

```diff
@@ -0,0 +1,266 @@
+- **requiredDeposit**() — returns the deposit required to perform a randomness request
+- **getRequestStatus**(*uint256* requestId) — returns the request status of a given randomness request
+- **getRequest**(*uint256* requestId) — returns the request details of a given randomness request
+- **requestLocalVRFRandomWords**(*address* refundAddress, *uint256* fee, *uint64* gasLimit, *bytes32* salt, *uint8* numWords, *uint64* delay) — request random words generated from the parachain VRF
+- **requestRelayBabeEpochRandomWords**(*address* refundAddress, *uint256* fee, *uint64* gasLimit, *bytes32* salt, *uint8* numWords) — request random words generated from the relay chain BABE consensus 
+- **fulfillRequest**(*uint256* requestId) — fulfill the request which will call the consumer contract method [`fulfillRandomWords`](#:~:text=rawFulfillRandomWords(uint256 requestId, uint256[] memory randomWords)). Fees of the caller are refunded if the request is fulfillable
+- **increaseRequestFee**(*uint256* requestId, *uint256* feeIncrease) — increases the fee associated with a given randomness request. This is needed if the gas price increases significantly before the request is fulfilled
+- **purgeExpiredRequest**(*uint256* requestId) — removes a given expired request from storage and transfers the request fees to the caller and the deposit back to the 
+
+Where the inputs that need to be provided can be defined as:
+
+- **requestId** - the ID of the randomness request
+- **refundAddress** - the address receiving the left-over fees after the fulfillment
+- **fee** - the amount to set aside to pay for the fulfillment
+- **gasLimit** - the gas limit to use for the fulfillment
+- **salt** - a string that is mixed with the randomness seed to obtain different random words
+- **numWords** - the number of random words requested, up to the maximum number of random words
+- **delay** - the number of blocks that must pass before the request can be fulfilled. This value will need to be between the minimum and maximum number of blocks before a local VRF request can be fulfilled
+- **feeIncrease** - the amount to increase fees by
+
+### Constants {: #constants }
+
+The interface includes the following constants:
+
+- **MAX_RANDOM_WORDS** - the maximum number of random words being requested. Set to {{ networks.moonbase.randomness.max_random_words }} words
+- **MIN_VRF_BLOCKS_DELAY** - the minimum number of blocks before a request can be fulfilled for local VRF requests. Set to {{ networks.moonbase.randomness.min_vrf_blocks_delay }} blocks
+- **MAX_VRF_BLOCKS_DELAY** - the maximum number of blocks before a request can be fulfilled for local VRF requests. Set to {{ networks.moonbase.randomness.max_vrf_blocks_delay }} blocks
+- **REQUEST_DEPOSIT_AMOUNT** - the deposit amount needed to request random words. There is one deposit per request. Set to {{ networks.moonbase.randomness.req_deposit_amount }} Wei
```

Comment:
deposit in DEV? or provide (as in the Learn page) the data in a tabbed table for all networks?